ReadMe.md

```diff
@@ -1,18 +1,107 @@
-# Hello-Microverse_Project
 
-Hello-Microverse_project is a beginner level project at Microverse, that requires requires the student to display hello on a web page with simple styling. 
 
-# Built With
-HTML
-CSS
+# 📗 Table of Contents
 
-# Key Features
-1. Linter for HTML and CSS: checks errors and organise code
-2. gitignore: to keep some directories from being committed
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
```

Comment:
- [ ] **Good job so far 👏 👏** However, As suggested by the previous reviewer, It would be better to update your file using [this updated template](https://github.com/microverseinc/readme-template) 😉, 

   Currently, the **⭐️ Show your support** and **📝 License** sections are missing in your README, Kindly try to add them and remember to customize the content 🙏